### ISSUE TITLE
Problem with navigation view helper (setPartial not working properly)

### ISSUE THREAD

**Issue by wesperinteractive** [zendframework/zend-view#91]
Setting the module when calling setPartial has absolutely no effect.

Consider the following code:

```
<?php echo $this->navigation('navigation')
    ->menu()
    ->setMinDepth(0)
    ->setPartial(array('partial/menu.phtml', 'Application'));
 ?>
```

renderPartialModel function (https://github.com/zendframework/zend-view/blob/master/src/Helper/Navigation/Menu.php#L751) will return the $partialHelper with the following parameters:

return $partialHelper($partial[0], $model); // 'partial/menu.phtml' and $model

If we call the setPartial function with a string

```
<?php echo $this->navigation('navigation')
    ->menu()
    ->setMinDepth(0)
    ->setPartial('partial/menu.phtml');
 ?>
```

the same happens:

return $partialHelper($partial, $model); // 'partial/menu.phtml' and $model

Why is this a problem? If you have modules with two different menu templates saved under the same name (partial/menu.phtml), then the modules loading order will decide which template is used.

**Comment by froschdesign** [comment]
> Setting the module when calling setPartial has absolutely no effect.

Right, because the `module` option is an obsolete option parameter.

> If you have modules with two different menu templates saved under the same name (partial/menu.phtml), then the modules loading order will decide which template is used.

You can specified the module name with the `partial` parameter:

```php
echo $this->navigation('navigation')->menu()->setPartial('application/partial/menu');
```
The same for the partial helper:
```php
echo $this->partial('application/partial/my-partial');
```

**Comment by froschdesign** [comment]
#92 removes the `module` option from the DocBlocks.

**Comment by froschdesign** [comment]
Closing: Fixed with #92 and the correct usage of a partial is documented.
